src/package/Microsoft.TestPlatform.CLI/Microsoft.TestPlatform.CLI.nuspec

```diff
@@ -55,6 +55,12 @@
     <file src="netcoreapp3.1\vstest.console.deps.json" target="contentFiles\any\netcoreapp3.1" />
     <file src="netcoreapp3.1\vstest.console.runtimeconfig.json" target="contentFiles\any\netcoreapp3.1" />
 
+    <file src="netcoreapp3.1\runtimes\unix\lib\netcoreapp2.1\System.Security.Principal.Windows.dll" target="contentFiles\any\netcoreapp3.1\runtimes\unix\lib\netcoreapp2.1" />
+    <file src="netcoreapp3.1\runtimes\win\lib\netcoreapp2.0\System.Security.AccessControl.dll" target="contentFiles\any\netcoreapp3.1\runtimes\win\lib\netcoreapp2.0" />
+    <file src="netcoreapp3.1\runtimes\win\lib\netcoreapp2.1\System.Security.Principal.Windows.dll" target="contentFiles\any\netcoreapp3.1\runtimes\win\lib\netcoreapp2.1" />
+    <file src="netcoreapp3.1\runtimes\win\lib\netstandard2.0\Microsoft.Win32.Registry.dll" target="contentFiles\any\netcoreapp3.1\runtimes\win\lib\netstandard2.0" />
+    <file src="netcoreapp3.1\Microsoft.Win32.Registry.dll" target="contentFiles\any\netcoreapp3.1\lib\netstandard2.0" />
```

Comment:
adding the missing dll that vstest.console is complaining about, but that change looks very dubious.